rid/op/mcmc_run.py

```diff
@@ -0,0 +1,127 @@
+from typing import List, Optional, Dict
+from pathlib import Path
+import numpy as np
+from dflow.python import (
+    OP,
+    OPIO,
+    OPIOSign,
+    Artifact,
+    Parameter,
+    BigParameter
+)
+from rid.utils import save_txt, set_directory
+from rid.mcmc.walker import Walker,my_hist2d
+from rid.select.model_devi import test_ef
+from rid.common.tensorflow.graph import load_graph
+try:
+    import tensorflow.compat.v1 as tf
+    tf.disable_v2_behavior()
+except ImportError:
+    import tensorflow as tf
+from rid.constants import mcmc_cv_name
+    
+# kinetic enery in eV
```

Comment:
Is it possible to move these constants to rid.constants, so that we can maintian a unified constants file?